portfolio/src/main/java/com/google/sps/servlets/ListServlet.java

```diff
@@ -0,0 +1,45 @@
+import com.google.cloud.translate.Translate;
+import com.google.cloud.translate.TranslateOptions;
+import com.google.cloud.translate.Translation;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet responsible for listing comments. */
+@WebServlet("/list-comments")
+public class ListServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query("Comment");
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+    String languageCode = request.getParameter("language");
+    List<String> comments = new ArrayList<String>();
+    for (Entity entity : results.asIterable()) {
+      String title = (String) entity.getProperty("text");
+      Translate translate = TranslateOptions.getDefaultInstance().getService();
+    Translation translation =
+        translate.translate(title, Translate.TranslateOption.targetLanguage(languageCode));
+    String comment = translation.getTranslatedText();
+      comments.add(comment);
```

Comment:
Make sure you keep indentation consistent.
```suggestion
      Translation translation =
        translate.translate(title, Translate.TranslateOption.targetLanguage(languageCode));
      String comment = translation.getTranslatedText();
        comments.add(comment);
```